SPECS/xcp-ng-linstor.spec

```diff
@@ -0,0 +1,22 @@
+Summary: Install all LINSTOR dependencies
+Name: xcp-ng-linstor
+Version: 1.0.0
```

Comment:
This version number could be used to version the whole XOSTOR thing. So as it's still a beta, wouldn't it be better to pick a version lower than 1?

Reply:
No preference on my side. :)

Reply:
Then I suggest 0.5.0 or 0.6.0